### PR TITLE
Create a list of companies/organizations contributing to Kubeflow

### PR DESCRIPTION
**Pull request by jlewi**
Start a file to keep track of organizations participating in Kubeflow.

* With Kubeflow around the corner we want to be sure to aknowledge the
  companies/organizations who are participating in Kubeflow.

* I think starting a file in source control is the best way to manage
  this over time.

/cc @ddysher @mattf @ewilderj @aronchick @elsonrodriguez @inc0 @willingc

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/63)

### CODE REVIEW

**Comment by ddysher** [comment]
Do we have plan to add it to a README or is there any tools to convert it to a website?

File: member_organizations.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This file provides information about companies/organizations
+# that would like to be aknowledged as contributing/participating in
```
Comment (willingc) [inline member_organizations.yaml]: s/aknowledged/acknowledged

File: member_organizations.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#
+# Optional fields
+#    url - Url for the organization 
+#    hiringForKubeflow - true/false whether the orgniazation is hiring for Kubeflow related
+#      positions
+#    hiringUrl - A URL for potential job applicants.
+#
+# We do not infer organizations based on affiliation in members.yaml because
+# some members may be contributing as individuals and not part of a deeper involvement
+# with Kubeflow for their company/organization.
```
Comment (inc0) [inline member_organizations.yaml]: I'd add note that this document is entirely voluntary and companies might contribute as they like without adding footprint there. A lot of companies just contributes one or two patches total and, while being very valid contributions, are reluctant to make commitments

Comment (jlewi) [reply member_organizations.yaml]: Isn't that covered by the first line of the file? "This file provides information about companies that would like to be acknowledged"
So if companies don't want to be acknowledged then they shouldn't add themselves.

**Comment by jlewi** [comment]
Fixed; PTAL.

I also added instructions to our contributing.md.

@ddysher Yes we should auto-generate a page to be included in our website acknowledging contributors but that doesn't exist yet. The immediate goal is just being able to get an accurate list before Kubecon so that we can manually create slides that don't leave anyone out.

**Comment by jlewi** [comment]
@ewilderj @aronchick @inc0 can someone please "/lgtm"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,8 +61,34 @@ the following tags:
...
 
-# Owners
+  * The only **required** field is your GitHub id
+  * This is a **prerequisite** for joining the GitHub org
+
+## Companies/Organizations
+
+If you would like your company or organization to be acknowledged for contributing to
+Kubeflow or participatng in the community (being a user counts) please send a PR
+adding the relevant info to[member_organizations.yaml](https://github.com/kubeflow/community/blob/master/member_organizations.yaml).
```
Comment (ewilderj) [inline CONTRIBUTING.md]: add space between `to` and `[member_organizations.yaml]`

**Pull request merged**